tests/tests/bitasset_tests.cpp

```diff
@@ -542,6 +544,455 @@ BOOST_AUTO_TEST_CASE( hf_890_test )
    }
 }
 
+class bitasset_evaluator_wrapper : public asset_update_bitasset_evaluator
```

Comment:
Interesting approach, good idea!
This might be a way to distinguish unit tests (like this one) from integration tests (what we usually do using PUSH_TX). In the future.